data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg

```diff
@@ -94,9 +94,13 @@
     [side]
         side=1
         controller=human
-        recruit=Thug,Poacher,Spearman,Bowman,Gryphon,Dwarvish Fighter,Dwarvish Thunderer,Dwarvish Ulfserker,Dwarvish Scout,Footpad
+        # Tallin only asked the mages for their gryphons at the end of the previous scenario, so don't add any more of their units here:
+        recruit=Thug,Poacher,Spearman,Bowman,Gryphon,Dwarvish Fighter,Dwarvish Thunderer,Dwarvish Ulfserker,Dwarvish Scout,Footpad,Fencer,Heavy Infantryman
```

Comment:
Fencers would look really odd in the Tallin's army of former ragged peasants. I think it's better to assume that the mages kept them to themselves as personal guardsmen.

Reply:
I actually initially added the fencers earlier in S06a "Old Friend", when Tallin and friends get the dwarves to forge new arms and armor for them and train them, the reasoning being that swords are newly forged, and that sometimes maps in NR get so crowded that units with skirmisher are necessary. The only reason I'm explicitly listing them in that scenario, too, is because since the recruit list is already explicitly listed there, it should at least be kept consistent with whatever units were previously added to it. If fencers don't seem ragged enough, perhaps some other unit with skirmisher could be created to fill that role instead?

Reply:
@kabachuha That argument applies equally to heavy infantry as well.

Reply:
> @kabachuha That argument applies equally to heavy infantry as well.

My same reply about the fencers applies to heavy infantry as well, except for the part about having skirmisher

Reply:
> If fencers don't seem ragged enough, perhaps some other unit with skirmisher could be created to fill that role instead?

What about the outlaw rogues? They have the skirmish ability and we already have thugs and footpads on the list, so I think they fit the band.

Reply:
> What about the outlaw rogues? They have the skirmish ability and we already have thugs and footpads on the list, so I think they fit the band.

Rogues are level 2, though, and it would seem odd to have those as the one "raw" level 2 unit available for direct recruitment...

Reply:
Well, if players could recruit Outlaw thiefs, they would be able to advance them into Rogue skirmishers. I don't think it takes too much experience.

Reply:
I dunno, I don't really feel like thieves fit, though... maybe earlier in the campaign when they had less options available to them, but at this point in the campaign they have other loyalist units and should be able to forge the high-quality swords necessary for fencers to be able to use

Reply:
They still have thugs, poachers and footpads for recruit though.

Reply:
Fencing is not as much about the armor as about the training. I doubt Tallin's men could have mastered fencing in a couple of years the campaign takes place even if they had the best steel.

If they lend them from mages, one or two loyal fencer/HI units may be sufficient and not too breaking story- and gameplaywise.

Reply:
How about a campaign-specific unit tree? Possibly liminal, using recolored thief sprites (different hair color), and somewhere between a Fencer and an L1 Outlaw?

Among the close-knit community of survivors, thieves had little place. However, a fighting style of nimble footwork suited some more than heavy armor, and no-one would begrudge backstabbing, when said back belonged to a orc.

Reply:
> How about a campaign-specific unit tree? Possibly liminal, using recolored thief sprites (different hair color), and somewhere between a Fencer and an L1 Outlaw?
> 
> Among the close-knit community of survivors, thieves had little place. However, a fighting style of nimble footwork suited some more than heavy armor, and no-one would begrudge backstabbing, when said back belonged to a orc.

That might be a good longer-term goal, but it seems a bit out-of-scope for this PR...

Reply:
So @Dalas121 recently had a proposal to give skirmisher to the Dwarvish Scout line; if that goes through, I could revert this part without having to invent any new unit...